src/cmd/ign.hh

```diff
@@ -18,28 +18,30 @@
 #ifndef IGNITION_GUI_IGN_HH_
 #define IGNITION_GUI_IGN_HH_
```

Comment:
`ign.hh` is installed, so we need to tick-tock it.

Reply:
I reverted the changes to ign.hh in 4e87255